amethyst_input/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "amethyst_input"
+version = "0.1.0"
+authors = ["Rhuagh <seamonr@gmail.com>"]
```

Comment:
I hate to be that guy, but given that I authored the majority of the functionality in this crate can you stick me as an additional author here?

Reply:
Wow, did I forget that? Soooorrry :~(